hdt-lib/src/triples/BitmapTriplesIterators.cpp

```diff
@@ -881,10 +886,10 @@ bool ObjectIndexIterator::canGoTo()
 
 void ObjectIndexIterator::goTo(unsigned int pos)
 {
-    if(minIndex+pos>maxIndex) {
-	throw std::runtime_error("Cannot goto beyond last element");
+    if(pos>maxIndex) {
+        throw std::runtime_error("Cannot goto beyond last element");
```

Comment:
Let's maybe be more descriptive and add `pos` and `maxIndex` to the error message.